catkin/install.yaml

```diff
@@ -2,4 +2,9 @@
   name: ros-setup
 
 - type: target
-  name: python-catkin-tools
+  default:
+    name: python3-catkin-tools
```

Comment:
I think this one still requires `python3-osrf-pycommon` no? The 'common' workaround in noetic.

Reply:
Yep, this is a target dependency. In the target, it is included.

Reply:
Ah, correct!